src/hooks/useClaimAll.ts

```diff
@@ -103,19 +96,16 @@ export function useClaimAll() {
       totalWeight = totalWeight.add(tx.weight);
     }
 
-    // The fix causes problems and confusion for stakers because rewards are not restaked,
-    // second thing is that developers are unable to stake.
-    // Need to change approach
     // Temporary disable restaking reward to avoid possible claim errors.
-    // const dappStakingRepository = container.get<IDappStakingRepository>(
-    //   Symbols.DappStakingRepository
-    // );
-    // const ledger = await dappStakingRepository.getLedger(senderAddress.value);
+    const dappStakingRepository = container.get<IDappStakingRepository>(
```

Comment:
This code block should be removed (everything that is commented in main)

Reply:
Oh, I see. Could you help fix and push commits? I think we still have to work on this branch.

Reply:
@impelcrypto  It looks like that most changes from https://github.com/AstarNetwork/astar-apps/pull/624 have been reverted during merge. Do you know how this happened?

Reply:
Ah, because I followed [this](https://github.com/AstarNetwork/astar-apps/pull/640) branch for this file... then I should have followed main branch.  

https://github.com/AstarNetwork/astar-apps/blob/d2173449bda845910d93b8c6303808d7c00b70bb/src/hooks/useClaimAll.ts

Reply:
Ok. I will fix this

Reply:
Thank you!